src/components/map-component.tsx

```diff
@@ -186,7 +170,7 @@ export class MapComponent extends BaseComponent<IProps, IState>{
           zoomControl={true}
           doubleClickZoom={true}
           scrollWheelZoom={true}
-          dragging={!showCrossSectionSelector}
+          dragging={!hasErupted}
```

Comment:
This is the only line I find strange.  

Do we really want to prevent dragging if the volcano has erupted?